cmake/Deps.cmake

```diff
@@ -8,7 +8,6 @@ set(DEPS_DOWNLOAD_DIR "${DEPS_BUILD_DIR}/downloads")
 
 set(DEPS_CMAKE_ARGS
   -D CMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-  -D CMAKE_C_STANDARD=99
```

Comment:
TODO(dundargoc): investigate if we should nuke c99 across the board

Reply:
Opted to only build treesitter with c11 when using wasmtime, this way we can postpone the discsussion of bumping to c11 to a later time.